consensus/polybft/consensus_runtime.go

```diff
@@ -760,17 +765,15 @@ func (c *consensusRuntime) calculateUptime(currentBlock *types.Header, epoch *ep
 	endBlock := getEndEpochBlockNumber(epoch.Number, c.config.PolyBFTConfig.EpochSize)
 
 	blockHeader := currentBlock
-	validators := epoch.Validators
-
-	var found bool
+	blockExists := false
```

Comment:
Why do you need this outside the loop?

Reply:
I have just renamed that variable and initialized it. The reason for having it outside from the loop is `blockHeader` instance, because we need that one to be initialized outside of for loop and therefore we need to move that `blockExists` outside as well.

FTR: https://github.com/0xPolygon/polygon-edge/pull/1069/files#diff-e5b2e93010ff84c3777aef136e0834d710dd22861ca62ffbab9d5baa12a52b3cR795